src/patterns/ec2-app.ts

```diff
@@ -393,11 +393,16 @@ export class GuEc2App {
     }
 
     if (!props.monitoringConfiguration.noMonitoring) {
-      new Gu5xxPercentageAlarm(scope, "Alarm", {
+      new Gu5xxPercentageAlarm(scope, {
         app,
         loadBalancer,
         ...props.monitoringConfiguration,
       });
+      new GuUnhealthyHostsAlarm(scope, {
+        app,
+        targetGroup,
+        snsTopicName: props.monitoringConfiguration.snsTopicName,
```

Comment:
I feel like this should become an account wide expectation that we lookup in Parameter Store, similar to the logging stream.

Reply:
I think the difficulty with doing this here is that a single account may host apps maintained by multiple teams. Sharing a logging stream is probably OK, but sharing an alerts topic could lead to a lot of noise. In recent times even Deploy Tools falls into this category (we are probably spamming Client-Side with a lot of alerts they don't care about right now). I think our long-term solution needs to look up appropriate targets based on tags (perhaps going via Anghammarad, which already has this capability).